### DISCUSSION TITLE
Buscar CEP pelo logradouro

### DISCUSSION THREAD

**Discussion by edysegura** [BrasilAPI/BrasilAPI#247] [Ideas]
Olá!

Parabéns pela iniciativa!! Gostaria de sugerir um endpoint para retornar os possíveis CEP informando um logradouro.

![image](https://user-images.githubusercontent.com/5324414/74394413-55c74f00-4deb-11ea-984b-bd97405acd96.png)

![image](https://user-images.githubusercontent.com/5324414/74394454-68da1f00-4deb-11ea-9c71-2920eec71bba.png)

**Comment by filipedeschamps** [comment]
Ah que show!!!! E agora ótima questão: isso primeiro deveria ser implementado em um endpoint separado no BrasilAPI, com cache e tudo mais, e depois utilizado no cep-promise ou outra lib, correto?
